README.md

```diff
@@ -115,3 +102,22 @@ Here is a complete table of the provided `form-*` classes for reference:
 | `select[multiple]`        | `form-multiselect` |
 | `[type='checkbox']`       | `form-checkbox`    |
 | `[type='radio']`          | `form-radio`       |
+
+### Using only base styles or only classes
+
+Although we recommend thinking of this plugin as a "form reset" rather than a collection of form component styles, in some cases our default approach may be too heavy-handed, especially when integrating this plugin into existing projects.
+
+For situations where the default strategy of generating both the base styles *and* classes doesn't work well with your project, you can use `strategy` option to control whether or not form styling is _opt-in_ or applied globally:
+
+```js
+// tailwind.config.js
+plugins: [
+  require("@tailwindcss/forms")({
+    strategy: 'base', // or strategy: 'class',
+  }),
```

Comment:
Is this maybe more clear? 🤔 
```suggestion
  require("@tailwindcss/forms")({
    strategy: 'base', // only generate global styles
    strategy: 'class', // only generate classes
  }),
```

Reply:
I went back and forth on this one before pushing the code :joy: